test/suites/dev/moonbase/test-fees/test-fee-multiplier-xcm.ts

```diff
@@ -87,34 +95,6 @@ describeSuite({
 
     it({
       id: "T02",
```

Comment:
Reasoning behind removal:

Original T02 was removed because it intended to test logic not related with XCM activity. The substrate primitives it tested was not implemented by us and is already tested through rust tests.

Reply:
https://github.com/paritytech/polkadot-sdk/blob/9064fb4d6dea751c537e33cb7c0a78c2510a1dce/substrate/bin/node/cli/tests/fees.rs#L38